app/domain/authentication/security/validate_role_can_access_webservice.rb

```diff
@@ -18,13 +18,14 @@ module Security
     ) do
 
       def call
-        # No checks required for default conjur authn
+        validate_account_exists
+        validate_role_is_defined
+
+        # No further checks required for default conjur authn
         return if default_conjur_authn?
 
-        validate_account_exists
         validate_webservice_exists
-        validate_user_is_defined
-        validate_user_has_access
+        validate_role_has_access
```

Comment:
I wonder if moving from user to role does not create confusion for the future devs